src/components/WorkspaceDetails/OverviewTab.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { OverviewPage, DetailsItem, K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
+
+type OverviewTabProps = {
+  loaded: boolean;
+  resource: K8sResourceCommon;
+  error: { message: string; status: number };
```

Comment:
@kdoberst can you see if the SDK exposes a type for this? If not - we probably need to create an issue to expose a type for it on the Core teams backlog.

Reply:
The error is LoadError within the SDK and it is not exported outside.